FParsecCS/FParsecCS-NETCore.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>FParsecCS</AssemblyName>
+    <RootNamespace>FParsec</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
+    <DefineConstants>LOW_TRUST;AGGRESSIVE_INLINING</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Properties\*.cs" />
+    <Compile Include="*.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FSharp.Core" Version="4.*" />
```

Comment:
Please change this to:

`<PackageReference Update="FSharp.Core" Version="4.3.4" />`

If you eventually publish a new version that uses these project files, it is recommended to target and explicit best-lowest version of FSharp.Core for maximum reach.

The .NET Core SDK contains a default reference to FSharp.Core for app developers (who generally want the latest version in their apps). The above syntax will basically say, "No, use this version instead" when you build your library, and it will ensure that consumers of this library will always reference the correct FSharp.Core when they build themselves.